login/core/validations/SignUpValidation.js

```diff
@@ -0,0 +1,13 @@
+const dbValidation = require('./DbValidation.js');
+const mailValidation = require('./MailValidation.js');
+const passValidation = require('./PasswordValidation.js');
+const loginValidation = require('./LoginValidation.js');
+
+function SignUpValidation(registrationRequest, response) {
```

Comment:
Maybe the issue is with the name, just call the function SignUp and put it into controllers folder. Tbh, look through your folder structure and the usual folder structure of node projects, and then make changes accordingly.

Reply:
I read about it but need more experience at foldering
I found this statment https://medium.com/codebase/structure-of-a-nodejs-api-project-cdecb46ef3f8 and trying make like there